modules/auxiliary/scanner/ssh/juniper_backdoor.rb

```diff
@@ -43,11 +43,12 @@ def initialize(info = {})
   def run_host(ip)
     factory = ssh_socket_factory
     ssh_opts = {
-      port:         rport,
-      auth_methods: ['password', 'keyboard-interactive'],
-      password:     %q{<<< %s(un='%s') = %u},
-      proxy: factory,
-      :non_interactive => true
```

Comment:
this was one hash that had a mix of the styles

Reply:
Yeah, yuck. And they're still all symbol keys, so they COULD have used the newer syntax. Oh well. Hashrockets it is!